### DISCUSSION TITLE
ConversationHandler in Telegram Channels

### DISCUSSION THREAD

**Discussion by rrrusst** [python-telegram-bot/python-telegram-bot#3814] [Q&A] [answered]
Is there a way to get ConversationHandler to work in channels, or if there is an alternative to ConversationHandler for channels? The purpose is to allow channel admins to configure the bot from within the channel, which in turn, changes the way the bot sends messages in the channel. It works in private and group chats without problem, but does nothing in channels. The bot can be used in multiple groups and channels and each can have their own user configurations.

If I used a MessageHandler (with Regex filter) as the entry point of ConversationHandler, nothing happens when I type the command in the channel. However, if I added the MessageHandler (with Regex filter) separately to the application, my command works (I see the desired output by the bot), but it stops there and no further conversation occurs.

**Comment by Bibo-Joshi** [comment, marked as answer]
Hi. Please read the note in the [docs of CH](https://docs.python-telegram-bot.org/en/stable/telegram.ext.conversationhandler.html#conversationhandler). Having the channel admins interact with the bot in the channel would be like watching the news reporter discussing the time schedule with the manager while on air - doesn't make much sense from a tv watcher/channel subscriber perspective ;)

**Reply by rrrusst** [reply]
Hmm I see. I guess what you described is generally what's considered normal when it comes to channel usage. I'll just stick with group/private chats for my bot then. Thanks for the clarification!
